gui/src/components/s3/bucket-creation.vue

```diff
@@ -384,6 +383,10 @@ export default class CortxBucketCreation extends Vue {
         text: "Name",
         value: "name",
         sortable: false
+      },
+      { 
+        text: "Action", 
```

Comment:
Can we use `i18n.t("common.action")` here instead of plain text?

Same in other places.

Reply:
resolved above review comment